src/components/Events.tsx

```diff
@@ -0,0 +1,48 @@
+import type { FC } from "react";
+
+export const mockEvents = [
+  {
+    id: 1,
+    name: "Vendy",
+    sport: "Basketball",
+  },
+  {
+    id: 2,
+    name: "Lukas",
+    sport: "Football",
+  },
+  {
+    id: 3,
+    name: "Honza",
+    sport: "Tennis",
+  },
+  {
+    id: 4,
+    name: "Vendy",
+    sport: "Box",
+  },
+  {
+    id: 5,
+    name: "Lukas",
+    sport: "Skydiving",
+  },
+  {
+    id: 6,
+    name: "Johana",
+    sport: "Jogging",
+  },
+];
+
+export const Events: FC = () => {
```

Comment:
Let's go with React.FC. Just personal preference. I did find that the most common on projects I've stumbled upon.

```suggestion
export const Events: React.FC = () => {
```